docs/content/docs/control_plane_health_probes.md

```diff
@@ -122,9 +122,9 @@ If any health probes are consistently failing, perform the following steps to id
     ```console
     $ osm mesh list
     
-    MESH NAME   NAMESPACE      CONTROLLER PODS
-    osm         osm-system     osm-controller-5494bcffb6-qpjdv
-    osm2        osm-system-2   osm-controller-48fd3c810d-sornc
+    MESH NAME   NAMESPACE      CONTROLLER PODS                   VERSION
+    osm         osm-system     osm-controller-5494bcffb6-qpjdv   vX.Y.Z
```

Comment:
Since the other properties are sample values, I'd drop in a sample value for the version as well (ex. v0.8.2)